docker/kratos/oathkeeper/id_token.jwks.json

```diff
@@ -0,0 +1,19 @@
+{
+  "keys": [
```

Comment:
We need to override this with a secure new key.

Reply:
Here is how to do that with Java: https://connect2id.com/products/nimbus-jose-jwt/examples/jwk-generation

Reply:
Or we can just use Ory's tools:
```go
package main

import (
	"encoding/json"
	"log"
	"os"

	"github.com/ory/hydra/jwk"
)

func main() {
	gen := jwk.RS256Generator{
		KeyLength: 2048,
	}
	jwks, err := gen.Generate("", "sig")
	if err != nil {
		log.Fatal(err)
	}
	jsonbuf, err := json.MarshalIndent(jwks, "", "  ")
	if err != nil {
		log.Fatal("failed to generate JSON: %s", err)
	}
	os.Stdout.Write(jsonbuf)
}
```